x-pack/plugins/ml/public/explorer/explorer.html

```diff
@@ -94,17 +126,24 @@
           <div ng-if="!showViewBySwimlane" class="text-center visError euiText">
             <div class="item top"></div>
             <div class="item">
-              <h4 class="euiTitle euiTitle--small">No {{swimlaneViewByFieldName}} influencers found</h4>
+              <h4
+                class="euiTitle euiTitle--small"
+                i18n-id="xpack.ml.explorer.noInfluencersFoundTitle"
+                i18n-default-message="No {swimlaneViewByFieldName} influencers found"
+                i18n-values="{ swimlaneViewByFieldName }"
+              ></h4>
             </div>
             <div class="item bottom"></div>
           </div>
 
         </div>
 
         <div ng-show="annotationsData.length > 0">
-          <span class="panel-title euiText">
-            Annotations
-          </span>
+          <span
+            class="panel-title euiText"
+            18n-id="xpack.ml.explorer.annotationsTitle"
+            i18n-default-message="Annotations"
+          ></span>
```

Comment:
This doesn't get rendered correctly, the string "Annotations" doesn't end up as inner HTML/text of the span element.

![image](https://user-images.githubusercontent.com/230104/50634957-97cb6380-0f50-11e9-88b8-d70af983c4cf.png)

![image](https://user-images.githubusercontent.com/230104/50634947-8b470b00-0f50-11e9-9b03-0d1ea8192b54.png)

Note the string "Annotation" visible in the screenshot is the string from the table column header, not the "Annotations" string this code is about.